utils/snow.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/ava-labs/avalanchego/snow/validators"
 	"github.com/ava-labs/avalanchego/snow/validators/validatorstest"
 	"github.com/ava-labs/avalanchego/utils/constants"
-	"github.com/ava-labs/avalanchego/utils/crypto/bls"
+	"github.com/ava-labs/avalanchego/utils/crypto/bls/signers/local"
```

Comment:
Should I rename this to `localSigner` or something similar?

Reply:
Preferrably `localsigner` all in lowercase